core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -795,22 +795,9 @@ class ZkUtils(val zkClient: ZkClient,
     * client updates the path stat.version gets incremented
     */
   def getSequenceId(path: String, acls: java.util.List[ACL] = DefaultAcls): Int = {
-    try {
+      makeSurePersistentPathExists(path)
```

Comment:
@ijuma  Agreed. Thanks for the above observation. Updated the PR with required changes.